src/main/emme/toolbox/export/export_data_loader_network.py

```diff
@@ -93,9 +93,10 @@ def __init__(self):
         self.main_directory = os.path.dirname(project_dir)
         self.base_scenario_id = 100
         self.traffic_emmebank = os.path.join(project_dir, "Database", "emmebank")
-        self.transit_emmebank = os.path.join(project_dir, "Database_transit", "emmebank")
+        # self.transit_emmebank = os.path.join(project_dir, "Database_transit", "emmebank")
+        self.transit_emmebank_dict = {}
```

Comment:
Do we still have "Database_transit' for transit_emmebank? What's the updated path of transit_emmebank?

Reply:
There are separate emmebanks for each time period now, as Database_transit_EA, Database_transit_AM, etc.